lib/puppet/pops/model/model_meta.rb

```diff
@@ -100,7 +100,7 @@ class UnfoldExpression < UnaryExpression; end
   # An assignment expression assigns a value to the lval() of the left_expr.
   #
   class AssignmentExpression < BinaryExpression
-    has_attr 'operator', OpAssignment, :lowerBound => 1
+    has_attr 'operator', String, :lowerBound => 1
```

Comment:
Yes, in the Pcore model this is an Enum. See [ast.pp, line 83](https://github.com/puppetlabs/puppet/pull/5648/commits/0f0c8778e214ff5e5cc11fbee12a912c6addb5a0#diff-56bfa3148b52eaab040806043811b7bcR83)